apps/src/applab/applab.js

```diff
@@ -782,6 +775,40 @@ Applab.init = function(config) {
   return loader;
 };
 
+async function initDataTab(levelOptions) {
+  if (levelOptions.dataTables) {
+    Applab.storage.populateTable(levelOptions.dataTables).catch(outputError);
+  }
+  if (levelOptions.dataProperties) {
+    Applab.storage.populateKeyValue(
+      levelOptions.dataProperties,
+      () => {},
+      outputError
```

Comment:
Why the difference in error handling? (i.e. here it's outputError, below it's console.log)

Reply:
fixed!